### PR TITLE
Move certifications to common

### PR DESCRIPTION
**Pull request by jcscottiii**
Add interface for certification to common.
Addresses https://github.com/opencontrol/compliance-masonry/pull/212/files#r74363210 by using `GetStandard` instead of `GetStandards`

A future PR will include the mock for it once the `common` package is cleaned up as talked about in #213

### CODE REVIEW

**Comment by jcscottiii** [comment]
@afeld This should be ready

File: commands/docs/gitbook/gitbookSummaries.go [hunk 1]
```diff
@@ -21,22 +21,29 @@ func (openControl *OpenControlGitBook) buildStandardsSummaries() (string, *map[s
...
+			familySummaryMap[newFamily] += exportLink(controlFullName, componentLink)
+			summary += "\t" + exportLink(controlFullName, filepath.Join("standards", componentLink))
 		}
-		controlFullName := fmt.Sprintf("%s: %s", controlKey, controlName)
-		familySummaryMap[newFamily] += exportLink(controlFullName, componentLink)
-		summary += "\t" + exportLink(controlFullName, filepath.Join("standards", componentLink))
-	})
+	}
 	return summary, &familySummaryMap
 }
```
Comment (afeld) [inline commands/docs/gitbook/gitbookSummaries.go]: This is a pretty large function...can we split it up so it's easier to follow?

**Comment by afeld** [comment]
Just one thing—looks good otherwise!

File: commands/docs/gitbook/gitbookSummaries.go (hunk 1 above)
Comment (jcscottiii) [inline commands/docs/gitbook/gitbookSummaries.go]: it's hard to break up this logic completely. i added some wrappers to some repeated logic to DRY it up and added comments.

**Comment by jcscottiii** [comment]
Also solves #224

File: lib/certifications/versions/1_0_0/certification_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package certification
```
Comment (afeld) [inline lib/certifications/versions/1_0_0/certification_test.go]: Ditt about the `certification_test` package.

Comment (jcscottiii) [inline lib/certifications/versions/1_0_0/certification_test.go]: will do

File: lib/certifications/certification_test.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package certifications
```
Comment (jcscottiii) [inline lib/certifications/certification_test.go]: will do

File: commands/docs/gitbook/gitbookSummaries.go [hunk 4]
```diff
@@ -6,37 +6,95 @@ import (
...
+	name := ""
+	for i, fileNamePart := range fileNameParts {
+		if i == len(fileNameParts)-1 {
+			// if it's the last part, no need to append the hyphen
+			name = fmt.Sprintf("%s%s", name, fileNamePart)
+		} else {
+			// if there is more left, append the hyphen
+			name = fmt.Sprintf("%s%s-", name, fileNamePart)
+		}
+	}
```
Comment (jcscottiii) [inline commands/docs/gitbook/gitbookSummaries.go]: doh. yep!

File: commands/docs/gitbook/gitbookSummaries.go (hunk 1 above)
Comment (jcscottiii) [inline commands/docs/gitbook/gitbookSummaries.go]: done

File: commands/docs/gitbook/gitbookSummaries.go (hunk 4 above)
Comment (jcscottiii) [inline commands/docs/gitbook/gitbookSummaries.go]: Done

File: lib/certifications/versions/1_0_0/certification_test.go (hunk 2 above)
Comment (jcscottiii) [inline lib/certifications/versions/1_0_0/certification_test.go]: Done

**Pull request merged**